### PR TITLE
Fix imports and semantics in HTML in `templates/bootstrap/views/index.php`

### PR DESCRIPTION
**Pull request by WinterSilence**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ❌
| Breaks BC?    | ❌
| Fixed issues  |

### CODE REVIEW

File: templates/bootstrap/views/index.php [hunk 1]
```diff
@@ -1,38 +1,41 @@
...
-use yii\apidoc\models\ClassDoc;
-use yii\apidoc\models\InterfaceDoc;
-use yii\apidoc\models\TraitDoc;
+use yii\apidoc\helpers\ApiMarkdown;
 
-/* @var $types ClassDoc[]|InterfaceDoc[]|TraitDoc[] */
-/* @var $this yii\web\View */
-/* @var $renderer \yii\apidoc\templates\html\ApiRenderer */
+/**
+ * @var yii\apidoc\models\ClassDoc[]|yii\apidoc\models\InterfaceDoc[]|yii\apidoc\models\TraitDoc[] $types
```
Comment (arogachev) [inline templates/bootstrap/views/index.php:6]: I'd leave explicit import of classes used in PHPDoc. Especially with unions like this it's more readable.

File: templates/bootstrap/views/index.php [hunk 2]
```diff
@@ -1,38 +1,41 @@
...
+ * @var string|null $readme
+ * @var yii\web\View $this
+ */
 
+/** @var yii\apidoc\templates\bootstrap\ApiRenderer $renderer */
 $renderer = $this->context;
 
+ksort($types);
```
Comment (arogachev) [inline templates/bootstrap/views/index.php:14]: Seems like classes are already correctly sorted at index page.

**Review by arogachev — COMMENTED** [review]
:ok_hand:

File: templates/bootstrap/views/index.php (hunk 2 above)
Comment (arogachev) [reply templates/bootstrap/views/index.php:14]: Nope, my bad, it was just moved higher.

File: templates/bootstrap/views/index.php (hunk 1 above)
Comment (WinterSilence) [reply templates/bootstrap/views/index.php:6]: @arogachev PHPDoc created not to be human readable

**Comment by WinterSilence** [comment]
@arogachev P.S. Изменять чужие PR - плохая практика. Правки должны исходить от автора PR, чтобы избежать проблем вроде: "@WinterSilence все сломал в PR #289", при том что критические изменения внесены не автором

**Pull request merged**